project-management/src/main/java/life/qbic/projectmanagement/application/purchase/OfferDTO.java

```diff
@@ -0,0 +1,12 @@
+package life.qbic.projectmanagement.application.purchase;
+
+/**
+ * <b><record short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+public record OfferDTO(boolean signed, String fileName, byte[] content) {
```

Comment:
When using arrays you have to overwrite equals and hashcode as by default array content is not compared. This only compares the arrays and might lead to misleading equals calls.